src/k8s-configuration/azext_k8s_configuration/vendored_sdks/v2021_09_01/aio/operations/__init__.py

```diff
@@ -8,10 +8,20 @@
 
 from ._extensions_operations import ExtensionsOperations
 from ._operation_status_operations import OperationStatusOperations
+from ._cluster_extension_type_operations import ClusterExtensionTypeOperations
```

Comment:
CI problem should be introduced by this line. May I ask why do you want to modify this file?

Reply:
I did not make that change I do not know why it is included in this.

Reply:
OK, please revert this change in your PR

Reply:
I have reverted that change and now all checks are passing. Thank you for finding the issue!